v2/spanner-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/SpannerToSourceDb.java

```diff
@@ -614,7 +655,9 @@ public static PipelineResult run(Options options) {
                         options.getShardingCustomClassName(),
                         options.getShardingCustomParameters(),
                         options.getMaxShardConnections()
-                            * shards.size()))) // currently assuming that all shards accept the same
+                            * shards.size()))) // currently assuming that all mySqlShards
+            // accept the same// currently assuming
```

Comment:
It seems like an additional comment appended here.

Reply:
resolved @bharadwaj-aditya